src/MessagingBubble/MessagingBubble.less

```diff
@@ -19,3 +19,14 @@
   .text--line-height-4();
   white-space: pre-wrap;
 }
+
+// For mobile/tablet
+@media only screen and (max-width: @breakpointM) {
+  .MessagingBubble--Message--Own {
+    .text--line-height-1();
```

Comment:
on figma mocks it looks like font-size is 14px and line-height is 16px? So seems line-height 1 wouldnt match

Reply:
Maybe font-size is what needs to be fixed?